### PR TITLE
Introduce new version of Ktor CLI generator

### PR DESCRIPTION
**Pull request by Stexxe**
The initial version allows generating a project with a specified name, has a verbose mode, and logs low-level errors.

### CODE REVIEW

File: cmd/ktor/main.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		projectName := utils.CleanProjectName(args.CommandArgs[0])
+		projectDir, err := filepath.Abs(projectName)
+
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to determine project directory for project %s\n", projectName)
+			os.Exit(1)
+		}
+
+		err = generate.Project(client, verboseLogger, projectDir, projectName)
+		if err != nil {
```
Comment (e5l) [inline cmd/ktor/main.go:72]: Please consider extracting this in a separate function. I would also extract error mappings in the separate collection

File: cmd/ktor/main.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+		}
+
+		if err == nil {
+			hasGlobalLog = true
+			defer logHandle.Close()
+		}
+	}
+
+	switch *args.Command {
+	case "new":
```
Comment (e5l) [inline cmd/ktor/main.go]: it would be great to have enum like consts here

Comment (e5l) [reply cmd/ktor/main.go]: I would also split the main in different small functions with a clear name

File: internal/app/cli/usage.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func WriteUsage(w io.Writer) {
+	fmt.Fprintf(w, "Ktor is a tool mainly for generating Ktor projects.\n\n")
```
Comment (e5l) [inline internal/app/cli/usage.go:10]: @vnikolova, could you please check the readme?

**Review by e5l — APPROVED** [review]
LGTM in general! please consider bringing some structure and waiting for the message review

File: internal/app/cli/usage.go (hunk 3 above)
Comment (vnikolova) [reply internal/app/cli/usage.go:10]: Is there supposed to be a README file for how to run the project locally as well?

Comment (Stexxe) [reply internal/app/cli/usage.go:10]: Not yet

**Review by e5l — APPROVED** [review]
looks better

**Pull request merged**